### PR TITLE
Add forms with status complete to the unsent filter

### PR DESCRIPTION
**Pull request by ShivamPokhriyal**
This'll make forms with status COMPLETE and UNSENT appear same to the user.

### CODE REVIEW

**Comment by shubham1g5** [comment]
@ShivamPokhriyal  Think completed forms should only be added in the Submitted Forms filter to correctly represent the semantics here.

**Comment by ShivamPokhriyal** [comment]
> @ShivamPokhriyal Think completed forms should only be added in the Submitted Forms filter to correctly represent the semantics here.

Not sure, since user can't quarantine submitted form. Moreover it represents forms that are sent to server. But completed are unprocessed forms and we should give user ability to quarantine these forms. Though user will get an option to delete. 
 
And if we add it to Submitted filter, then we'll have to add it to submitted and pending filter as well. 

@shubham1g5

**Comment by shubham1g5** [comment]
@ShivamPokhriyal Ahh yes,  I got confused meaning of `STATUS_SAVED` . This looks good to me Thanks!

File: app/src/org/commcare/activities/FormRecordListActivity.java [hunk 1]
```diff
@@ -99,7 +99,7 @@
         /**
          * Processed and Pending
```
Comment (shubham1g5) [inline app/src/org/commcare/activities/FormRecordListActivity.java]: Can we change this comment to `Submitted and Pending` instead ?

**Pull request merged**